Makefile

```diff
@@ -1,6 +1,7 @@
-HELM != helm
+HELM ?= helm
 
 gen-expected:
+	${HELM} dependency update .
```

Comment:
This will be long for every little changes...
You can add it in a separate step with a timestamps file and a dependency on Chart.yaml ...